README.md

```diff
@@ -1,27 +1,35 @@
-# ReactAdmin Enterprise Edition action
+# React Admin Enterprise Edition action
 
-This action sets up your ReactAdmin Enterprise Edition authentication by creating a `.npmrc` file (or an `.yarnrc.yml` file if you use Yarn v2 or more) and adding your secret token that you can find on your [ReactAdmin Enterprise Edition setup page](https://registry.marmelab.com/setup) to use [private modules](https://marmelab.com/ra-enterprise/#private-modules).
+This action sets up your package manager (`npm` or `yarn`) to use the [React Admin Enterprise Edition](https://marmelab.com/ra-enterprise/) private registry, which is required to download the [private modules](https://marmelab.com/ra-enterprise/#private-modules).
+
+It works by creating a `.npmrc` file (or a `.yarnrc.yml` file if you use Yarn v2 or more) and adding the CI authentication token found on the [React Admin Enterprise Edition setup page](https://registry.marmelab.com/setup).
```

Comment:
I used the terms `CI authentication token` since this is what the token is called in the https://registry.marmelab.com/setup page.